src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_blocking.cpp

```diff
@@ -34,6 +34,22 @@ void BrgemmBlocking::move_new_memory_buffer(snippets::lowered::LinearIR& linear_
         const auto wsp_it = linear_ir.find(wsp_expr);
         linear_ir.move(wsp_it, brgemm_it);
     }
+    return std::prev(brgemm_it);
+}
+
+LinearIR::constExprIt BrgemmBlocking::get_loop_begin_pos(LinearIR& linear_ir, const LinearIR::constExprIt& brgemm_it) {
+    auto loop_begin_it = brgemm_it;
+    const auto& brgemm_expr = *brgemm_it;
+    const auto brgemm = ov::as_type_ptr<ov::intel_cpu::BrgemmCPU>(brgemm_expr->get_node());
```

Comment:
Note that this pass may (and will, in case TPP) be used not only for `BrgemmCPU` ops.
Also, keeping in mind the ARM activities, it looks like we need a more future-proof solution.
Not saying we need to do it in this PR, but we definitely need to think about it now to avoid strong dependencies on current OneDNN's Brgemm interface, and also to understand what kind of generic interface we need.

Reply:
Good point, let's discuss it offline

Reply:
I think we have to cover both op::Brgemm (will be in case of TPP) and BrgemmCPU cases like it's done in the [run(...)](https://github.com/openvinotoolkit/openvino/blob/110d83e57c564a5e6c350faa3f754ac28fdff2e5/src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/brgemm_blocking.cpp#L77-L81)

Reply:
Done, thanks!